pkg/model/harbor/model.go

```diff
@@ -25,7 +24,6 @@ func (s Severity) String() string {
 }
 
 var severityToString = map[Severity]string{
-	SevNone:     "None",
 	SevUnknown:  "Unknown",
```

Comment:
None does exist as a CVSS severity level (of the score is 0): see https://nvd.nist.gov/vuln-metrics/cvss

Reply:
Indeed it's defined in CVSS v3 but we're not supposed to return VulnerabilityItems with Severity `None`. Also Trivy does not return packages with Severity equal to `None`.

It was there because Clair returned all packages (with and without vulns), but in the Scanner Adapter spec we're just concerned with vulnerable packages.